Nik

Nikoleta-v3 opened the pull request:
A few sentences from me. Mainly describing the PD and few things on the ZD strategies.

drvinceknight committed:
Merge pull request #15 from Axelrod-Python/vk

Progress towards first draft

drvinceknight committed:
Add boxplot visualisation.

Also: improve the colormap for the median plots.

drvinceknight committed:
Add a table of linear regression coefficients.

This is neat (I think) as it shows numerically that the N=2 case has
very little to do with an actual population dynamic.

drvinceknight committed:
Update plots.

drvinceknight committed:
Add exact moran process to validation script.

drvinceknight committed:
Prepare to run 1, N-1 case.

drvinceknight committed:
Revert symmetric case.

drvinceknight committed:
Remove exact from validation plot.

The run time is unreasonable.

drvinceknight committed:
Start refactoring for different initial population.

drvinceknight committed:
Readjust to include invasion/resistance.

drvinceknight committed:
Update results.

drvinceknight committed:
Add code to numerically find relative fitness.

Also remove the Noisy data from the cleaning step.

drvinceknight committed:
Change title: numerical -> empirical

drvinceknight committed:
Write a final data set with all data.

drvinceknight committed:
Add a `main.csv` data that makes all analysis easier.

drvinceknight committed:
Add more data.

drvinceknight committed:
Change a number of styles of plots and writing.

Modified the format from horizontal to vertical and I also used a
correlation matrix (which was essentially what I was doing before).

drvinceknight committed:
More analysis and writing:

- Add discussion about numerical inversion;
- Add plots of relative fitness for validation strategy pairs
- Particular section for N=2

Various other changes.

drvinceknight committed:
Update data.

drvinceknight committed:
Make possible to get p_1 when p_N/2 incomplete.

drvinceknight committed:
Add some discussion and references.

drvinceknight committed:
Minor reorganisation of files.

drvinceknight committed:
Add discussion to resistance section.

drvinceknight committed:
Add tables showing rank vs size.

drvinceknight committed:
Minor adjustments and run with new data.

drvinceknight committed:
Rerun with data for N=11, i=1

Nikoleta-v3 committed:
Add literature.

An introduction to the PD.
Two sentences about emergence of cooperation study
and two sentences about the ZD.

Nikoleta-v3 committed:
Merge branch 'vk' of github.com:Axelrod-Python/axelrod-moran into vk

drvinceknight commented on main.tex:
```diff
@@ -53,6 +53,15 @@
 
 \section{Introduction}\label{sec:introduction}
 
+The Prisoner's Dilemma (PD)~\cite{Tucker1983} is a fundamental two person game 
```
Reference the original Rand paper perhaps? (Not sure what this reference is).

drvinceknight commented on main.tex:
```diff
@@ -101,17 +110,24 @@ \section{Introduction}\label{sec:introduction}
+looked at in a theoretic framework for a small subset of strategies. 
+The subset included memory one strategies, strategies that recall the events 
+of the previous round only. In \cite{Press2012}, a set of memory one 
+strategies, the zero determinant strategies, have been introduced. The ZD\@,
```
were introduced.

drvinceknight commented on main.tex:
```diff
@@ -53,6 +53,15 @@
+the game are defined by the matrix $\begin{pmatrix} R & S \\ T & P
+\end{pmatrix}$, where $T > R > P> S$ and $2R > T + S$. The PD is a one 
+round game, but is commonly studied in a manner where the prior outcomes 
+matter, the game then extends to its ’iterated’ form, the Iterated Prisoner's 
```
``iterated'`

drvinceknight commented on main.tex:
```diff
@@ -407,7 +423,7 @@ \section{Validation}\label{sec:validation}
 
 Figure~\ref{fig:comparison_stochastic} shows the fixation probabilities for
 stochastic strategies. These are no longer a good match which highlights the
-weakness of the analytical formulae that relies on the average payoffs. A
```
formulae is the plural

drvinceknight reviewed (changes requested):
Add your name to the authors.  

```
\author{Marc Harper \and Vincent Knight} % TODO Authors?
```

Nikoleta-v3 committed:
Vine corrections.

I have also included my name in the authors. Thank you once again
for the opportunity.

I decided to sign my name as 'Nikoleta E. Glynatsi` from now on
